lms/djangoapps/course_home_api/outline/views.py

```diff
@@ -253,37 +249,18 @@ def get(self, request, *args, **kwargs):  # pylint: disable=too-many-statements
 
             enable_proctored_exams = course_overview.enable_proctored_exams
 
-            if COURSE_GOALS_NUMBER_OF_DAYS_GOALS.is_enabled():
-                if (is_enrolled and ENABLE_COURSE_GOALS.is_enabled(course_key)):
-
-                    course_goals = {
-                        'selected_goal': None,
-                        'weekly_learning_goal_enabled': True,
-                    }
-                    selected_goal = get_course_goal(request.user, course_key)
-                    if selected_goal:
-                        course_goals['selected_goal'] = {
-                            'days_per_week': selected_goal.days_per_week,
-                            'subscribed_to_reminders': selected_goal.subscribed_to_reminders,
-                        }
-            else:
-                # Only show the set course goal message for enrolled, unverified
-                # users in a course that allows for verified statuses.
-                is_already_verified = CourseEnrollment.is_enrolled_as_verified(request.user, course_key)
-                if not is_already_verified and has_course_goal_permission(request, course_key_string,
-                                                                          {'is_enrolled': is_enrolled}):
-                    course_goals = {
-                        'goal_options': valid_course_goals_ordered(include_unsure=True),
-                        'selected_goal': None
+            if (is_enrolled and ENABLE_COURSE_GOALS.is_enabled(course_key)):
+                course_goals = {
+                    'selected_goal': None,
```

Comment:
Above this in this method, we already set this value. And set `goal_options`, which isn't used anymore, right?

Reply:
So we do! I missed that. I'll clean it up.